blocks/header/header.js

```diff
@@ -9,22 +9,22 @@ import
   decorateAnchors,
 } from '../../scripts/scripts.js';
 
-function decorateSocial(social) {
-  social.classList.add('social');
-  social.innerHTML = social.innerHTML.replace(/\[social\]/, '');
-  social.querySelectorAll(':scope>ul>li').forEach((li) => {
-    const a = li.querySelector('a');
-    a.setAttribute('target', '_blank');
-    if (a.innerHTML.includes('linkedin')) {
-      a.setAttribute('aria-label', 'LinkedIn');
-    } else if (a.innerHTML.includes('twitter')) {
-      a.setAttribute('aria-label', 'Twitter');
-    } else if (a.innerHTML.includes('facebook')) {
-      a.setAttribute('aria-label', 'Facebook');
-    } else if (a.innerHTML.includes('youtube')) {
-      a.setAttribute('aria-label', 'YouTube');
-    }
+async function decorateWebsitePicker(websitePicker) {
+  websitePicker.classList.add('website-picker');
+  websitePicker.innerHTML = websitePicker.innerHTML.replace(/\[websites\]/, '');
+  const title = 'Sunstar Websites';
+  websitePicker.querySelectorAll(':scope>ul>li').forEach((li) => {
+    li.classList.add('website-picker-item');
   });
+
+  const a = document.createElement('a');
```

Comment:
I am not sure if its needed or not because the dropdown is not an anchor on the sunstar.com

Reply:
Some elements in top-nav support redirection to different pages like Careers and global network and styles like font size are common for both elements i.e. clickable or unclickable, for a quick fix removed href from <a>